cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -86,4 +86,4 @@ EXAMPLES
            $ bal build <app-package-path>
 
        Build the package with additional GraalVM native image options.
-           $ bal build --graalvm --graalvm-build-options="--static -H:Name=hello-world"
+           $ bal build --graalvm --graalvm-build-options="--static -o <entire-file-path-with-name>"
```

Comment:
We cannot use the `-o` option again
Since this is an example for the `--graalvm-build-options` we may add this `--enable-monitoring` option instead of `-H:Name=hello-world` option
```suggestion
           $ bal build --graalvm --graalvm-build-options="--static --enable-monitoring"
```

Reply:
Got it. I'll fix these.